web/src/components/KebabMenu.tsx

```diff
@@ -0,0 +1,104 @@
+import React, { ReactElement, ReactNode, useState } from 'react'
+import styled from 'styled-components'
+
+import { UiDirectionType } from 'translations'
+
+import iconClose from '../assets/IconClose.svg'
+import iconKebabMenu from '../assets/IconKebabMenu.svg'
+import dimensions from '../constants/dimensions'
+import '../styles/KebabMenu.css'
+import { Portal } from './Portal'
+
+type KebabMenuProps = {
+  items: Array<ReactNode>
+  direction: UiDirectionType
+}
+
+const ToggleContainer = styled.div`
+  display: flex;
+  padding: 0 8px;
+  z-index: 32;
```

Comment:
well we have the app where we can use up to 1x, bottomActionSheet 2x, and Sidebar 3x with the components inside which may have different z-index. If there are any better suggestions please let me know

Reply:
Not sure what the used z-indeces are, but it is recommended to use 30, 40, 50 and so on, and when needing a value in between, it is best to use the "middel", so you always have options to put stuff between, because values are still available.

Reply:
well we use 10 1000, 2000 and so one. i've fine with sticking to 10x values and gonna change it